traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -229,6 +229,7 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.4, http.MethodPost, `user/login/oauth/?$`, login.OauthLoginHandler(d.DB, d.Config), 0, NoAuth, nil, 1415886009, noPerlBypass},
 		{1.1, http.MethodPost, `user/login/token(/|\.json)?$`, login.TokenLoginHandler(d.DB, d.Config), 0, NoAuth, nil, 402408841, perlBypass},
 		{1.1, http.MethodPost, `user/reset_password(/|\.json)?$`, login.ResetPassword(d.DB, d.Config), 0, NoAuth, nil, 2092914630, perlBypass},
+		{1.1, http.MethodPost, `users/register(/|\.json)?$`, login.RegisterUser, auth.PrivLevelOperations, Authenticated, nil, 1337, perlBypass},
```

Comment:
should we allow bypass given ```Prior to ATC version 4.0, this endpoint allowed for the registration of users with arbitrary
		:term:`Roles`. It now restricts the allowed values to identifiers for :term:`Roles` with at
		most the same permissions level as the requesting user.```

Reply:
Well the default is that it won't bypass, and it's up to administrators to eat that risk if they want to bypass it. I raised a concern about that (and other concerns) with the security team and it was decided that these didn't merit "vulnerability" status, so :man_shrugging:

Reply:
okie dookie 👍